docs/pages/guides/apple-privacy.mdx

```diff
@@ -0,0 +1,69 @@
+
+```json
+{
+  "expo": {
+    "name": "My App",
+    "slug": "my-app",
+    // other fields...
+    "ios": {
+      "privacyManifests": {
+        "NSPrivacyAccessedAPITypes": [
+          {
+            "NSPrivacyAccessedAPIType": "NSPrivacyAccessedAPICategoryUserDefaults",
+            "NSPrivacyAccessedAPITypeReasons": ["CA92.1"]
+          }
+        ]
+      }
+    }
+  }
+}
+```
+
+<ConfigReactNative>
+
+You can include an iOS privacy manifest in a bare Expo app by creating a **PrivacyInfo.xcprivacy** file using Xcode and adding it to your iOS app target.
+Follow [this guide](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files) to create a **PrivacyInfo.xcprivacy** file.
+
+</ConfigReactNative>
+
+You can identify the `NSPrivacyAccessedAPITypes` and `NSPrivacyAccessedAPITypeReasons` values by looking at the [Apple Developer Documentation](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api).
+
+> **warning** Make sure you have updated your Expo SDK packages to latest versions for your SDK version using `npx expo install --fix`.
+
+### Including required reasons for Expo SDK packages and other 3rd party libraries
+
+As of now, Apple does not correctly parse all the **PrivacyInfo** files included by static CocoaPods dependencies (such as Expo SDK packages and other ecosystem packages). This means that you may need to include the required reasons for the APIs used by those dependencies in your app's **PrivacyInfo.xcprivacy** file or the configuration in your **app.json**.
+
+All Expo SDK packages that use "required reason" APIs file have a **PrivacyInfo** file included in the package directory. [Here's an example file](https://github.com/expo/expo/blob/main/packages/expo-application/ios/PrivacyInfo.xcprivacy) included with the `expo-application` package.
+
+You can usually identify the required reasons for the APIs used by other 3rd party libraries by checking if the package you intend to use has a **PrivacyInfo.xcprivacy** file in the **node_modules/package_name/ios** directory. If it does, you can check the `NSPrivacyAccessedAPITypes` and `NSPrivacyAccessedAPITypeReasons` values in that file and copy those values to your configuration.
+
+As an alternative, Apple notifies developers after they submit a build with missing privacy manifest files or specific reasons, so you can wait until you receive a notification email from Apple and then include the required reasons listed in the email in your app's **PrivacyInfo.xcprivacy** file (if you don't use [CNG](/workflow/continuous-native-generation/)) or the configuration in your **app.json** file.
+
+## Testing the Privacy Manifest
+
+You can test the privacy manifest by building your app and submitting it, either to regular review or to TestFlight external review. Apple will notify you via email within a few minutes of submitting if you're missing any required reasons for the APIs used by your app.
```

Comment:
```suggestion
You can test the privacy manifest by building your app and submitting it, either through App Store review process or to TestFlight's external review. Apple will email you within a few minutes of submitting if your app is missing any required reasons for the APIs used.
```

- We can be concise with the last sentence. Also, let's clarify the that the review process is for App Store.